packages/picasso/src/Alert/Alert.tsx

```diff
@@ -79,7 +92,19 @@ export const Alert = forwardRef<HTMLDivElement, Props>(function Alert(
           {children}
         </Typography>
       </Container>
-      {onClose && renderAlertCloseButton({ onClose })}
+      <Container inline flex>
+        {actions?.primary && (
+          <Button {...actions?.primary} variant='primary' size='small'>
+            {actions?.primary?.label}
+          </Button>
+        )}
+        {actions?.secondary && (
+          <Button {...actions?.secondary} variant='secondary' size='small'>
+            {actions?.secondary?.label}
+          </Button>
+        )}
```

Comment:
Maybe we could put these behind a util function or component, something like:

```ts
const renderActionButton = (variant: ButtonProps['variant'], props: ButtonAction) => (
  <Button {...props} variant={variant} size='small'>{props.label}</Button>
)
```

And then:
```suggestion
        {actions?.primary && renderActionButton('primary', actions.secondary)}
        {actions?.secondary && renderActionButton('secondary', actions.secondary)}
```